www/install.rst

```diff
@@ -5,51 +5,82 @@ Installation
+system-wide versus local environment use, and control. With pip or Anaconda's
+conda, you can control the package versions for a specific project to prevent
+conflicts. Conda also controls non-Python packages, like MKL or HDF5. System
+package managers, like ``apt-get``, install across the entire computer, often
+have older versions, and don't have as many available versions. Source
+compilation is much more difficult but is necessary for debugging and development.
+If you don't know which installation method you need or prefer, we recommend
+the Scientific Python Distribution `Anaconda <https://www.anaconda.com/download/>`_ .
 
 .. _distributions:
 
-Scientific Python Distributions (Recommended)
+Scientific Python Distributions (recommended)
 =============================================
 
-Python distributions provide the language itself along with the most commonly used packages and tools. These downloadable files require little configuration, work on almost all setups, and provide all the commonly used scientific python tools.
+Python distributions provide the language itself, along with the most commonly
+used packages and tools. These downloadable files require little configuration,
+work on almost all setups, and provide all the commonly used scientific python tools.
 
-`Anaconda <https://www.anaconda.com/download/>`_ works on Windows, Mac, and Linux, provides over 1,500 Python/R packages, and is used by over 15 million people. Anaconda provides a lite version, `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_, which still provides access to the *conda* package manager.
+`Anaconda <https://www.anaconda.com/download/>`_ works on Windows, Mac, and
+Linux, provides over 1,500 Python/R packages, and is used by over 15 million
+people. Anaconda provides a lite version,
+`Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_,
+which still provides access to the *conda* package manager.
 
 Other options include:
 
-* `Enthought Canopy <https://www.enthought.com/products/canopy>`_: Free and commercial versions; Supports Linux, Windows, and Mac; Includes an IDE.
-* `Python(x,y) <https://python-xy.github.io/>`_: A free distribution including scientific packages; based around the `Spyder IDE <https://www.spyder-ide.org/>`__; Windows and Ubuntu; Py2 only.
-* `WinPython <https://winpython.github.io>`_: Another free distribution including scientific packages and the Spyder IDE; Windows only, but more actively maintained and supports the latest Python 3 versions.
-* `Pyzo <http://www.pyzo.org/>`_: A free distribution based on Anaconda and the IEP interactive development environment; Supports Linux, Windows, and Mac.
-* `NeuroDebian repository <http://neuro.debian.net>`_ : Only for the Debian OS and geared toward neuroscientists.
+* `Enthought Canopy <https://www.enthought.com/products/canopy>`_: Free and
+commercial versions; supports Linux, Windows, and Mac; includes an IDE.
```

Comment:
These need an indent to line up with the first character (not the `*`) on the line above to render correctly. 

If you run `make html` you should see no warnings; this gives:
```
.../scipy.org/www/install.rst:41: WARNING: Bullet list ends without a blank line; unexpected unindent.
```